Makefile

```diff
@@ -75,7 +75,9 @@ install-provisioning-app: apply-provisioning-app-deploy
 
 ## Update OpenShift resources related to the Provisioning App service.
 apply-provisioning-app-deploy:
-	cd ods-provisioning-app/ocp-config && tailor apply --namespace ${NAMESPACE}
+	cd ods-provisioning-app/ocp-config && tailor apply --namespace ${NAMESPACE} is
+	../../ocp-scripts/import-image-from-dockerhub.sh --namespace ${NAMESPACE} --imagename ods-provisioning-app
```

Comment:
Path looks wrong - I think each line in the `Makefile` should be executed in a new shell, so this should be relative to the `Makefile` itself (as you have it in `apply-doc-gen-build`).